tensorflow/python/kernel_tests/matrix_triangular_solve_op_test.py

```diff
@@ -24,15 +24,17 @@
 class MatrixTriangularSolveOpTest(tf.test.TestCase):
```

Comment:
This is not being tested with GPU support.  You need to modify this line:

https://github.com/c0g/tensorflow/blob/e46db8b3e2d1efd5cf10ced68b15cbbd1d58a149/tensorflow/python/kernel_tests/BUILD#L216

from tf_py_test to cuda_py_test.